package.json

```diff
@@ -21,7 +21,8 @@
     "ngclipboard": "^2.0.0",
     "restangular": "^1.6.1",
     "underscore": "^1.8.3",
-    "wavesplatform-core-js": "git://github.com/wavesplatform/wavesplatform.core.js.git#master"
+    "wavesplatform-core-js": "git://github.com/wavesplatform/wavesplatform.core.js.git#master",
+    "opn": "^6.0.0"
```

Comment:
Add "opn" package, to make working the **open**ing wavesexplorer, by timeout, after start the server.
Just, like for new wavesexplorer.